packages/steppers/src/styled/StyledLabel.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled from 'styled-components';
+import {
+  getColor,
+  getLineHeight,
+  retrieveComponentStyles,
+  DEFAULT_THEME
+} from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'steppers.step_label';
+
+interface IStyledLabelProps {
+  isActive?: boolean;
+  isHorizontal?: boolean;
+}
+
+export const StyledLabel = styled.div.attrs<IStyledLabelProps>({
```

Comment:
@ginnywood I know we discussed applying horizontal margins to the labels, however, there could be some trade-offs (margins only noticed on Steppers with 10+ steps with long labels, effects of other parts of the layout, etc). Would you be down to sit together and see how we could make adjustments to the margins?

Reply:
Yes, grab some time with me!

Reply:
@ginnywood the steps now have horizontal spacing (`12px` each side). This was achieved by applying horizontal padding and a `min-width` on each step. The step label should have horizontal spacing now 👍.